pkg/migrations/sql/idempotent/base.sql

```diff
@@ -126,6 +126,24 @@ GRANT EXECUTE ON FUNCTION SCHEMA_CATALOG.unlock_metric_for_maintenance(int) TO p
 --is insignificant in practice.
 --
 --lock-order: metric table, data_table, series parent, series partition
```

Comment:
It looks like this comment was meant for the next function and you inserted a new function in here?